<launch>
<node pkg="rviz"  type="rviz"  name="rviz"  args="-d $(find gpm_slam)/rviz/front_end.rviz"></node>
<node pkg="gpm_slam"  type="front_end_node"  name="front_end_node"  output="screen"></node>
</launch>